pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -222,3 +232,53 @@ func (r *ReconcileJaeger) apply(jaeger v1.Jaeger, str strategy.S) error {
 
 	return nil
 }
+
+//addFinalizer will add this attribute to the Jaeger CR
+func (r *ReconcileJaeger) addFinalizer(jaeger *v1.Jaeger) error {
+	if len(jaeger.GetFinalizers()) < 1 && jaeger.GetDeletionTimestamp() == nil {
```

Comment:
Shouldn't you check `jaeger.GetDeletionTimestamp() == nil` && finalizer array does not contain our finalizer?

Reply:
well, taking into account that I'm sure that it won't be another finalizer, I think this is fine, but I can do the change to be more explicit.

Reply:
I changed this.